Resources/views/layout.html.twig

```diff
@@ -20,7 +20,7 @@ file that was distributed with this source code.
             {% block sonata_page_breadcrumb %}
                 <div class="row">
                     {% if sonata_seo_context is not defined %}
-                        {% set sonata_seo_context = 'homepage' %}
+                        {% set sonata_seo_context = 'page' %}
```

Comment:
yes, redirect with sylvaindeloux breadcrumb service dont change /{_locale} with the current used. i can add the request instead of sonata intl bundle service to detect local used. may you got a better idea ?